recipe-client-addon/lib/NormandyDriver.jsm

```diff
@@ -182,11 +182,11 @@ this.NormandyDriver = function(sandboxManager) {
 
     // Study storage API
     studies: {
-      create: sandboxManager.wrapAsync(StudyStorage.create, {cloneArguments: true}),
-      update: sandboxManager.wrapAsync(StudyStorage.update, {cloneArguments: true}),
-      get: sandboxManager.wrapAsync(StudyStorage.get, {cloneInto: true}),
-      getAll: sandboxManager.wrapAsync(StudyStorage.getAll, {cloneInto: true}),
-      has: sandboxManager.wrapAsync(StudyStorage.has),
+      start: sandboxManager.wrapAsync(AddonStudies.start, {cloneArguments: true, cloneInto: true}),
```

Comment:
Yeah... maybe I'm overly paranoid. But I'm assuming the installs don't warn in any way, so I wonder if we could restrict them in some way to mitigate risk (e.g. require that it's signed with the relevant key in the code here, irrespective of pref values, and/or require that it's downloaded from a specific domain, or something).